services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -416,17 +382,17 @@ public static Builder builder() {
    */
   @AutoValue.Builder
   public abstract static class Builder {
-
-    private List<Double[]> bearings = new ArrayList<>();
+    //TODO change List<Double> to a custom model or to a Pair
```

Comment:
Do you mean `List<List<Double>>` for `bearings`? If so, let's clarify in the comment.

Reply:
I mean it would be useful to change
`List<List<Double>>` -> `List<Bearing>` of `List<Pair>`